### ISSUE TITLE
The visualization of the segmentation results.

### ISSUE THREAD

**Issue by chenyzh28** [speedinghzl/pytorch-segmentation-toolbox#47]
I trained the model with 4 GPUs (GTX 2080) with batchsize 4, and the PSPNet achieves the mIOU of 76.42%. But I found the visualization results are not correct. Here is an example.
![image](https://user-images.githubusercontent.com/35193979/64757006-5a00ef00-d563-11e9-8212-a1095c30d2fd.png).
But the ground-truth label is like:
![image](https://user-images.githubusercontent.com/35193979/64757149-c2e86700-d563-11e9-805f-78d0b7142e0a.png)
So how can i modify the program to make it right?

**Comment by erichhhhho** [comment]
@chenyzh28 you can use the cityscapes colormap, this code uses a different color map. see get_palette()

**Comment by chenyzh28** [comment]
Thanks for your attention. But Where can I find the cityscapes colormap? Here is the original code of get_palette().
![image](https://user-images.githubusercontent.com/35193979/64757523-f5df2a80-d564-11e9-8d66-4c922ce78778.png)

**Comment by erichhhhho** [comment]
Refer to [here](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py)
Column Color

**Comment by chenyzh28** [comment]
Thank you very much! I have already corrected the colormap.
![image](https://user-images.githubusercontent.com/35193979/64758253-b0bbf800-d566-11e9-8639-a31aaa6e65c8.png)

**Comment by swjtulinxi** [comment]
hi,i have a question about why your results are just 76.42%,you have 4 GPU,but your results are not the same as paper 79 or 78.x
